.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "remote"]
 	path = remote
-	url = git@github.com:MusicConnectionMachine/api.git
+	url = https://github.com/MusicConnectionMachine/api.git
```

Comment:
@simonzachau ah right, I missed that commit.
wow... travis actually supports user keys only for paid plans... https://docs.travis-ci.com/user/private-dependencies/#User-Key

Yeah nevermind... this will be fixed in a new PR.